documentation/site/content/samples/azure-kubernetes-service/model-in-image.md

```diff
@@ -30,12 +52,12 @@ This sample demonstrates how to use the [WebLogic Kubernetes Operator](https://o
 Clone the [WebLogic Kubernetes Operator repository](https://github.com/oracle/weblogic-kubernetes-operator) to your machine. We will use several scripts in this repository to create a WebLogic domain. This sample was tested with v3.4.2, but should work with the latest release.
 
 ```shell
-$ git clone --branch v{{< latestVersion >}} https://github.com/oracle/weblogic-kubernetes-operator.git
-```
-```shell
-$ cd weblogic-kubernetes-operator
+$ cd $BASE_DIR
+$ git clone https://github.com/oracle/weblogic-kubernetes-operator.git
```

Comment:
Was it intentional to remove the "--branch" argument?

Reply:
Hello @rjeberhard , yes, "--branch" is removed intentionally. This sample self contains scripts, we can remove "--branch" to specify the specified script version.